src/plugins/console/public/styles/_app.scss

```diff
@@ -33,6 +33,47 @@
 .conApp__output {
   display: flex;
   flex: 1 1 1px;
+
+  .ace_badge {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding: 0 8px;
+    display: inline-block;
+    text-decoration: none;
+    border-radius: 3px;
+    border: solid 1px transparent;
+    white-space: nowrap;
+    vertical-align: middle;
+    cursor: default;
+    max-width: 100%;
+    text-align: left;
```

Comment:
I believe this style can be removed, as its left text alignment is already being inherited and applied.

```suggestion
```